ci_scripts/ci_post_clone.sh

```diff
@@ -77,6 +77,11 @@ if [[ "$target" == "release-package-build-"* ]]; then
     sed -i '' "s/REALM_HIDE_SYMBOLS = NO;/REALM_HIDE_SYMBOLS = YES;/" "$filename"
 fi
 
+# Xcode cloud currently doesn't have visionOS installed for Xcode 15.3
+if [[ "$CI_WORKFLOW" == release-package-build-vision*_15.3 ]]; then
```

Comment:
From experience this was causing some timeout issues when running the workflow. Is that still the case?